src/Analysis/GWASV2/Steps/ConfigureGWAS/ConfigureGWAS.jsx

```diff
@@ -255,10 +255,12 @@ const ConfigureGWAS = ({
               <div>Outcome Phenotype</div>
               <div>{outcome?.concept_name ?? outcome?.provided_name}</div>
             </div>
-            <div style={flexRow}>
-              <div>Final Population Size</div>
-              <div>{finalPopulationSize}</div>
-            </div>
+            {finalPopulationSizes.map((item, key) => (
+              <div key={key} style={flexRow}>
```

Comment:
**Nitpick**: Using an iterator as a react key I think is discouraged (https://dev.to/devcer/why-you-should-never-use-array-index-as-key-prop-in-your-react-applications-3m9o). If we certain the items are unique, I believe you could use an item as the key.  

ESLINT isn't complaining though so its probably OK

Reply:
we tried to fix and failed. The error complained `[object Object]` was not unique 🤔